modules/cli-administrator-cluster-management.adoc

```diff
@@ -5,10 +5,29 @@
 [id="cli-cluster-management-commands_{context}"]
 = Cluster management CLI commands
 
+== inspect
+
+Gather debugging information for a particular resource.
+
+[NOTE]
+====
+This command is experimental and may change without notice.
```

Comment:
s/may/might (all places)

>! may v
Use only to indicate permission. Otherwise, use “can” or “might.”